scalableminds/webknossos
# Location independent upload

Description of changes:
- data sets can be uploaded to every data store

Steps to test:
- upload a data set using the dashboard upload
- try to view / trace the data set to see if it was properly uploaded

Issues:
- fixes #1458

------


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1537/create?referer=github" target="_blank">Log Time</a>
